src/main/java/sx/blah/discord/modules/IModule.java

```diff
@@ -8,6 +8,9 @@
  * This class represents a Discord4J "module", implement it to create a new Module. When enabled, modules are registered
  * as event listeners, so your module can use {@link EventSubscriber} or
  * {@link IListener} to listen for events.
+ * Deployment: The Module Loader will first look for an attribute called "Discord4J-ModuleClass" in a jar's manifest which
```

Comment:
Add a mention of "Discord4J-ModuleClass" to the package-info.java in the modules package.